modules/runner/src/integrationTest/java/org/apache/ignite/internal/Cluster.java

```diff
@@ -135,12 +135,28 @@ public void startAndInit(int nodeCount) {
      * @param initParametersConfigurator Configure {@link InitParameters} before initializing the cluster.
      */
     public void startAndInit(int nodeCount, Consumer<InitParametersBuilder> initParametersConfigurator) {
+        startAndInit(nodeCount, defaultNodeBootstrapConfigTemplate, initParametersConfigurator);
+    }
+
+    /**
+     * Starts the cluster with the given number of nodes and initializes it.
+     *
+     * @param nodeCount Number of nodes in the cluster.
+     * @param nodeBootstrapConfigTemplate Node bootstrap config template to be used for each node started
+     *     with this call.
+     * @param initParametersConfigurator Configure {@link InitParameters} before initializing the cluster.
+     */
+    public void startAndInit(
+            int nodeCount,
+            String nodeBootstrapConfigTemplate,
```

Comment:
I propose to move this argument below.

Reply:
Why? Usually it is more convenient to have 'closure-like' arguments last in the arg list

Reply:
It just looks better to me when two (or more) methods/constructors have the same order of arguments.

Up to you.